docs/serverless-demo.md

```diff
@@ -69,40 +67,44 @@ mutation PausePlaybackMutation {
 
 You can begin [identifying your client traffic](https://www.apollographql.com/docs/graphos/metrics/client-awareness) by ensuring the proper headers are sent. GraphOS will automatically associate the graph usage with the identified client information:
 
-| Header name                    | Header value                                                                             |
-| ------------------------------ | ---------------------------------------------------------------------------------------- |
-| `apollographql-client-name`    | The name of your integration.                                                            |
-| `apollographql-client-version` | Increment this value with version numbers for your integration as its API usage changes. |
+| Header name                    | Header value                                                                        |
+| ------------------------------ | ----------------------------------------------------------------------------------- |
+| `apollographql-client-name`    | The name of your client.                                                            |
+| `apollographql-client-version` | Increment this value with version numbers for your client as its API usage changes. |
 
-> If you test out API operations in the [Explorer]({{ graph.url.explorer }}), add the `apollographql-client-name` and `apollographql-client-version` headers with `Apollo Explorer` and `main` (or whatever values you like).
+If you're using [Explorer]({{ graph.url.explorer }}), add the `apollographql-client-name` and `apollographql-client-version` headers with `Apollo Explorer` and `main` (or whatever values you like).
 
 If you have a client project, simply setting the `name` and `version` properties in the `ApolloClient` constructor will automatically add these header values to your requests for you. You can see how we set that in the Spotify Showcase Website [here](https://github.com/apollographql/spotify-showcase/blob/main/client/src/apollo.ts#L76):
 
-```graphql
+```js
 const client = new ApolloClient({
   link: new HttpLink({
-    uri: 'http://localhost:4000/graphql'
+    uri: 'http://localhost:4000/graphql',
   }),
   name: 'Spotify Showcase Website',
-  version: '1.0'
+  version: '1.0',
 });
 ```
 
-Once you've sent some traffic, head over to the [Clients tab]({{ graph.url.clients }}) and you should see the traffic being identified now 🎉
+Once you've sent some traffic, head over to the [Clients page]({{ graph.url.clients }}) and you should see the traffic being identified now 🎉
 
 ## Adding a new GraphQL API to GraphOS
 
-You can view all of the GraphQL APIs in the [subgraphs tab]({{  graph.url.subgraphs }}). Try adding SpaceX to the demo by using the "Add a subgraph" button. You'll use https://spacex-production.up.railway.app/ for the routing URL and it should be able to fetch the subgraph schema/
+You can view all of the GraphQL APIs in the Subgraphs page.
```

Comment:
Removed the link here until the shortcode is implemented https://apollographql.atlassian.net/browse/NEBULA-2562